sdk/relay/Azure.ResourceManager.Relay/src/Customized/RelayNamespaceData.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#nullable disable
+using System;
```

Comment:
Please run `eng\scripts\Export-API.ps1 relay` in sdk directory to update the API file.